noggin/templates/user-settings-password.html

```diff
@@ -6,10 +6,13 @@
   <div class="card-body">
     <h5 id="pageheading">{{ _("Change Password") }}</h5>
     <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-    <div class="form-group">{{ macros.with_errors(password_reset_form.current_password, tabindex="2")}}</div>
+    {# if the user has not enabled OTP, using_otp will be False. If False, we dont show the description in the field that talks about OTP #}
+    <div class="form-group">{{ macros.with_errors(password_reset_form.current_password, tabindex="2", show_description=using_otp)}}</div>
```

Comment:
Instead of adding `show_description` and setting it here, you could also change the field description directly in the controller, as in [the doc's example for choice fields](https://wtforms.readthedocs.io/en/stable/fields.html#wtforms.fields.SelectField). It would look something like this in the controller:
```python
if using_otp:
    form.current_password.description = ""
```